app/views/calendars/calendar_events/new.html.erb

```diff
@@ -1,3 +1,3 @@
 <h1>Create a new event</h1>
 
-<%= render 'calendars/calendar_events/form', calendar: @calendar, calendar_event: CalendarEvent.new %>
```

Comment:
This meant we never got error messages for any calendar events

Reply:
whoops!